src/main/java/cn/nukkit/command/defaults/KillCommand.java

```diff
@@ -60,7 +60,8 @@ public boolean execute(CommandSender sender, String commandLabel, String[] args)
                 for (Level level : Server.getInstance().getLevels().values()) {
                     for (Entity entity : level.getEntities()) {
                         if (!(entity instanceof Player)) {
-                            joiner.add(entity.getName());
+                            if (!entity.getName().isEmpty())
+                                joiner.add(entity.getName());
```

Comment:
What if we use the `entity.getSaveId()` when `getName()` is empty?

Also, how was getName() empty? Did you add a custom name with an empty string?